flatsurf/geometry/similarity_surface.py

```diff
@@ -2361,3 +2361,15 @@ def __hash__(self):
             h = h + 3*hash(edgepair)
         self._hash=h
         return h
+
+    def pyflatsurf_converter(self):
```

Comment:
I think I would prefer to call this `_to_pyflatsurf()` or make it a `_pyflatsurf` property even.

Any reason not to use a `cached_method` here instead? You can seed the cache if you have to set it from the outside. In particular, the content of a `cached_method` is not pickled which is probably what you want here?

Reply:
Name can be changed, I don't have any strong preference.

About caching, I wanted to be able to set the cache manually so that both
- `FlatsurfConverter(S).sage_flatsurf_surface() is S` for a `sage-flatsurf` surface `S`
- `FlatsurfConverter(T).pyflatsurf_surface() is T` for a `pyflatsurf` flat triangulation `T`

hold. `cached_method` could be used but it is not very convenient.

Reply:
Now a `_pyflatsurf` lazy attribute